daemon/daemon_windows.go

```diff
@@ -39,9 +39,10 @@ const (
 	windowsMaxCPUPercent = 100
 )
 
-// Windows doesn't really have rlimits.
+// Windows containers are much larger than Linux containers and each of them
+// have > 20 system processes which why it is better to restore them without parallelism.
 func adjustParallelLimit(n int, limit int) int {
-	return limit
+	return 1
```

Comment:
Did you overwrite your change? Wasn't this supposed to be a value calculated based on the number of CPUs/cores?

Reply:
My bad. Used another computer to do force push and forget pull first. Now it is correct on again.